app/controllers/rents_controller.rb

```diff
@@ -9,11 +9,25 @@ def create
     rent = Rent.create(rent_params)
     if rent.valid?
       render json: RentSerializer.new.serialize(rent).to_json, status: :created
+      book = Book.find(rent.book_id)
```

Comment:
You should use callbacks in Rent model, read this https://guides.rubyonrails.org/active_record_callbacks.html#callbacks-overview